clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -62,31 +66,49 @@
   private final Clock clock;
   private final KubectlJobExecutor jobExecutor;
 
-  private final String accountName;
-  @Getter private final List<String> namespaces;
-  @Getter private final List<String> omitNamespaces;
-  private final List<KubernetesKind> kinds;
-  private final Map<KubernetesKind, InvalidKindReason> omitKinds;
-  private final Map<KubernetesKind, InvalidKindReason> omitKindsComputed = new HashMap<>();
-  @Getter private final List<CustomKubernetesResource> customResources;
+  @Include private final String accountName;
+
+  @Include @Getter private final List<String> namespaces;
+
+  @Include @Getter private final List<String> omitNamespaces;
+
+  @Include private final List<KubernetesKind> kinds;
+
+  @Include List<String> omitKinds;
+
+  private final Map<KubernetesKind, InvalidKindReason> omitKindsComputed;
+
+  @Include @Getter private final List<CustomKubernetesResource> customResources;
+
+  @Include @Getter private final String kubectlExecutable;
+
+  @Include @Getter private final Integer kubectlRequestTimeoutSeconds;
+
+  @Include @Getter private final String kubeconfigFile;
+
+  @Include private final String kubeconfigFileHash;
+
+  @Include @Getter private final boolean serviceAccount;
+
+  @Include @Getter private final String context;
+
+  @Include @Getter private final boolean onlySpinnakerManaged;
+
+  @Include @Getter private final boolean liveManifestCalls;
+
+  @Include private final boolean checkPermissionsOnStartup;
 
-  @Getter private final String kubectlExecutable;
-  @Getter private final Integer kubectlRequestTimeoutSeconds;
-  @Getter private final String kubeconfigFile;
-  @Getter private final String kubeconfigFileHash;
-  @Getter private final boolean serviceAccount;
-  @Getter private final String context;
+  @Include @Getter private final List<KubernetesCachingPolicy> cachingPolicies;
 
-  @Getter private final boolean onlySpinnakerManaged;
-  @Getter private final boolean liveManifestCalls;
-  private final boolean checkPermissionsOnStartup;
-  @Getter private final List<KubernetesCachingPolicy> cachingPolicies;
+  @Include @JsonIgnore @Getter private final String oAuthServiceAccount;
 
-  @JsonIgnore @Getter private final String oAuthServiceAccount;
-  @JsonIgnore @Getter private final List<String> oAuthScopes;
+  @Include @JsonIgnore @Getter private final List<String> oAuthScopes;
 
-  @Getter private boolean metrics;
-  @Getter private final boolean debug;
+  @Include @Getter private boolean metrics;
```

Comment:
I think we could remove the `@Getter` on the `metrics` now to make it more clear that consumers shouldn't care.  (I think it's used in one test, but that test should probably just check the computed one.)